.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+### Describe your changes.
+* 
```

Comment:
I think these empty bullet points can be removed?

Reply:
I included them just to add a little bit of consistency of spacing between prompts (a place for the user to enter their response). Though I fully trust you if you think they should be removed (?). Just wanted to say that including them it wasn't an accident. @TimothyWillard

Reply:
That makes sense. I can implement that. Thanks